src/arrays/referencing_array_and_its_elements/hard/q4/PotentialGoodStudents.java

```diff
@@ -0,0 +1,27 @@
+package arrays.referencing_array_and_its_elements.hard.q4;
+
+/*
+Write a Java program that defines an array of doubles given the following array [ 73, 15, 71, 67, 5, 96, 91, 12, 5, 82 ],
```

Comment:
Same thing here. We need to provide some sort of input to the user so they can't just hardcode this question